src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/VedtakendringerService.kt

```diff
@@ -63,11 +63,11 @@ class VedtakendringerService(
         logger.info("Vedtak- og inntektsendringer ferdig")
     }
 
-    fun harNyeVedtak(response: InntektshistorikkResponse) = nyeVedtak(response)?.isNotEmpty() ?: false
+    fun harNyeVedtak(response: HentInntektListeResponse) = nyeVedtak(response)?.isNotEmpty() ?: false
 
     private fun lagreInntektsendringForPerson(
         forventetInntektForPerson: ForventetInntektForPerson,
-        response: InntektshistorikkResponse,
+        response: HentInntektListeResponse,
```

Comment:
Mulig jeg er tjukk i huet, men burde det være en klient-endring et sted her?

Reply:
hehe, jo egentlig, men vi var heldige her for vi hadde implementert klientkoden for det andre endepunktet både i personhendelse og proxy som lå igjen fra noe gammelt moro :D

Reply:
Da henger jeg med!